Traceback (most recent call last):
  File "C:\Users\CARLOS\AppData\Local\Temp\ipykernel_5056\3332378752.py", line 35, in train
    ("ord", ordinal_transformer, ordinales),
NameError: name 'ordinales' is not defined